COPYRIGHT

```diff
@@ -1,3 +1,4 @@
+2021 California Institute of Technology
```

Comment:
This copyright file is wrong -- I see that it still says LSST Corp even though LSST Corp assigned those over to AURA.

Reply:
Will fix

Reply:
Fixed